clms-angular/src/app/modules/dashboard/pages/dashboard-page/dashboard-page.component.html

```diff
@@ -18,6 +18,6 @@ <h1 class="text-3xl font-bold text-on-surface">My Courses</h1>
   </div>
   <div class="w-full h-full bg-surface-c flex flex-col p-5 gap-10 overflow-y-scroll">
     <div *ngIf="myAssignments.length === 0"> No Assignments available </div>
-    <app-assignments-list [assignments]="myAssignments"></app-assignments-list>
+    <app-assignments-list *ngIf="myAssignments.length > 0" [assignments]="myAssignments"></app-assignments-list>
   </div>
```

Comment:
*ngIf on this app-assignments-list lets us render this component only when we have assignments available from the network request